frontend/src/shared-components/about-details/about.details.jsx

```diff
@@ -17,8 +17,8 @@ export const AboutDetails = ({ imageSrc, name, type, text, socialLinks }) => {
           </h3>
           <p className={styles.aboutParagraph}>{text}</p>
           <ul className={styles.aboutSocialLinks}>
-            {socialLinks.map((link) => (
-              <a href={link.url}>
+            {socialLinks.map((link, index) => (
+              <a key={index} href={link.url}>
                 <img src={link.iconPath} alt="" />
               </a>
             ))}
```

Comment:
Naziv `AboutDetails` ne govori nista o tome za sta je komponenta namenjena. 
Citanjem implementacije se moze zakljuciti da je komponenta namenjena za podatke o autoru, ali to ne bi trebalo da bude neophodno da se radi samo kako bi se saznalo za sta je komponenta namenjena. 
Naziv komponente bi trebao da bude dovoljan indikator svrhe koju komponenta sluzi.
U ovom slucaju, bolji naziv bi bio `AboutAuthor` ili `AuthorDetails` ili `BlogAuthor` (ovaj poslenji odgovara nazivu koji se koristi na backend-u).